python/ray/air/config.py

```diff
@@ -307,15 +307,15 @@ class DatasetConfig:
         transform: Whether to transform the dataset with the fitted preprocessor.
             This must be enabled at least for the dataset that is fit.
             True by default.
-        use_stream_api: Whether the dataset should be streamed into memory using
-            pipelined reads. When enabled, get_dataset_shard() returns DatasetPipeline
-            instead of Dataset. The amount of memory to use is controlled
-            by `stream_window_size`. False by default.
-        stream_window_size: Configure the streaming window size in bytes.
-            A good value is something like 20% of object store memory.
-            If set to -1, then an infinite window size will be used (similar to
-            bulk ingest). This only has an effect if use_stream_api is set.
-            Set to 1.0 GiB by default.
+        max_object_store_memory_fraction [Experimental]: The maximum fraction
+            of Ray's shared-memory object store to use for the transformed
+            Dataset. The default value is -1, meaning that the preprocessed
+            dataset should be cached, which may cause spilling if its size is
+            larger than the object store's capacity. Pipelined ingest (all
+            other values, from 0 to 1) is experimental. Note that the absolute
+            memory capacity used is based on the object store capacity at
+            invocation time; this does not currently cover autoscaling cases
+            where the size of the cluster may change.
```

Comment:
Is `None` a valid value? If so, could we describe the behavior when `max_object_store_memory_fraction=None`

Reply:
I think it's okay not to document this since it follows the same convention as the other `DatasetConfig` args (`None` means default).